### PR TITLE
Store gitbase indexes in a volume

### LINKED ISSUE #65: Add persistence to gitbase index data
I haven't tested this, but I think with the current config the gitbase container will loose the index data after a `docker-compose down`. If it is stored in a volume this data will persist after a `down; up`

### PR DESCRIPTION
**Pull request by carlosms**
Fix #65

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ services:
         target: /opt/repos
         read_only: true
         consistency: delegated
+      - indexes:/var/lib/gitbase/index
```
Comment (smacker) [inline docker-compose.yml]: 1. space
2. let's call it `gitbase_indexes` or something like that. `indexes` is way too general name.

Comment (carlosms) [reply docker-compose.yml]: > space

what do you mean?

Comment (smacker) [reply docker-compose.yml]: ah. My bad. It is just below lines with syntax highlighted but this line not. But actually it's correct.

Comment (carlosms) [reply docker-compose.yml]: :+1: name changed

**Pull request merged**